src/content/resources/glossary.md

```diff
@@ -279,3 +279,49 @@ If not, the pattern _refutes_, or denies, the match.
 Refutable patterns appear in [_matching contexts_][].
 
 [_matching contexts_]: /language/patterns#matching
+
+## Subclass
+
+A _subclass_ is a class that inherits the implementation of another class by using the
+[`extends`](/language/extend) keyword, or by [mixin application](#mixin-application).
+
+```dart
+class A extends B {} // A is a subclass of B; B is the superclass of A. 
+
+class B1 with M {} // B1 is a subclass of M.
```

Comment:
`class B1 with M` is short for `class B1 extends Object with M`, so `B1`'s superclass is `Object with M`, whose own superclass is `Object`.

Reply:
Thanks, my mistake I meant to use Erik's original suggestion:

```dart
class B1 extends A with M {} // B1 has the superclass `A with M`, which has the superclass A.
```